src/utils/format.ts

```diff
@@ -15,27 +27,159 @@ function _getLocaleSymbols(): { thousands: string; decimals: string } {
 }
 
 const { thousands: THOUSANDS_SYMBOL, decimals: DECIMALS_SYMBOL } = _getLocaleSymbols()
-const DEFAULT_THOUSANDS_SYMBOL = ','
-const DEFAULT_DECIMALS_SYMBOL = '.'
+
+function _formatDecimalsForDisplay(decimalsToConvert: BigNumber, decimalSymbol: string = DEFAULT_DECIMALS_SYMBOL) {
+  const prefix = '0'
+  // e.g 00012366123
+  const decimalsWithoutIntegerOrSymbol = decimalsToConvert.toString(10).slice(2)
```

Comment:
Got confused at first.
This assumes `decimalsToConvert` will never have an integer part.

Maybe to be sure would be good to subtract any integer? https://mikemcl.github.io/bignumber.js/#int

```suggestion
  const decimalsWithoutIntegerOrSymbol = (decimalsToConvert.minus(decimalsToConvert.integerValue())).toString(10).slice(2)
```

Reply:
true, wasn't really accounting for negative numbers

Reply:
what is `.integerValue()`?

Reply:
nvm clicked the link like i probably should have done in the first place and am G2G